src/personal-settings.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @copyright Copyright (c) 2019 Gary Kim <gary@garykim.dev>
+ *
+ * @author Gary Kim <gary@garykim.dev>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import $ from 'jquery';
+import Axios from 'nextcloud-axios';
+
+$(document).ready(() => {
```

Comment:
In general I would prefer a tiny Vue settings component. Do you have experience with Vue? It's really simple. You can check out other code like https://github.com/nextcloud/twofactor_u2f/tree/master/src. You can create a settings component like https://github.com/nextcloud/twofactor_u2f/blob/master/src/components/PersonalSettings.vue and initialize it like in https://github.com/nextcloud/twofactor_u2f/blob/master/src/main-settings.js. Just let me know if you need help :)

Reply:
Thanks for the references. I'll try to figure it out.

Reply:
They are okay for simple tasks. But things usually just get more complex and the php templating does not scale.

Reply:
Personal settings have been changed to use Vue.